openassessment/xblock/static/js/src/lms/oa_staff_area.js

```diff
@@ -157,108 +198,148 @@
-                }
-            ).fail(function(errMsg) {
-                    $('#schedule_training_message', view.element).text(errMsg);
-                });
+            this.server.scheduleTraining()
+                .done(function(msg) {
+                    $('.schedule_training_message', view.element).text(msg);
+                }).fail(function(errMsg) {
+                $('.schedule_training_message', view.element).text(errMsg);
+            });
         },
 
         /**
-         Begins the process of rescheduling all unfinished grading tasks. This incdludes
-         checking if the classifiers have been created, and grading any unfinished
-         student submissions.
-
-         **/
+         * Begins the process of rescheduling all unfinished grading tasks.
+         * This includes checking if the classifiers have been created,
+         * and grading any unfinished student submissions.
+         */
         rescheduleUnfinishedTasks: function() {
             var view = this;
-            this.server.rescheduleUnfinishedTasks().done(
-                function(msg) {
-                    $('#reschedule_unfinished_tasks_message', view.element).text(msg);
-                }
-            ).fail(function(errMsg) {
-                    $('#reschedule_unfinished_tasks_message', view.element).text(errMsg);
-                });
+            this.server.rescheduleUnfinishedTasks()
+                .done(function(msg) {
+                    $('.reschedule_unfinished_tasks_message', view.element).text(msg);
+                }).fail(function(errMsg) {
+                $('.reschedule_unfinished_tasks_message', view.element).text(errMsg);
+            });
         },
 
         /**
-         Upon request, cancel the submission from grading pool.
-         **/
+         * Upon request, cancel the submission from grading pool.
+         */
         cancelSubmission: function(submissionUUID) {
             // Immediately disable the button to prevent multiple requests.
             this.cancelSubmissionEnabled(false);
             var view = this;
-            var sel = $('#openassessment__student-info', this.element);
-            var comments = sel.find('#staff-info__cancel-submission__comments').val();
-            this.server.cancelSubmission(submissionUUID, comments).done(
-                function(msg) {
+            var comments = $('.cancel_submission_comments', this.element).val();
+            this.server.cancelSubmission(submissionUUID, comments)
+                .done(function(msg) {
                     $('.cancel-submission-error').html('');
-                    $('#openassessment__staff-info__cancel__submission', view.element).html(msg);
-                }
-            ).fail(function(errMsg) {
+                    view.loadStudentInfo({expanded_view: 'final-grade'})
+                        .done(function() {
+                            $('.openassessment__staff-info__cancel__submission', view.element).html(msg);
```

Comment:
I put a note about this above-- do we really need to show the msg here? Aren't we reloading and showing the message in the final grade section instead?